bibtex-actions.el

```diff
@@ -536,15 +520,18 @@ With prefix, rebuild the cache before offering candidates."
 
 (defun bibtex-actions-run-default-action (keys)
   "Run the default action `bibtex-actions-default-action' on KEYS."
-  (funcall bibtex-actions-default-action
-           (if (stringp keys) (split-string keys " & ") keys)))
+  (let* ((keys-raw (if (consp keys) (cdr keys) keys))
+         (keys-list
+          ;; Make sure we split the crm-string we provide 'embark'.
+          (if (stringp keys-raw) (split-string keys-raw) " & " keys-raw)))
+    (funcall bibtex-actions-default-action keys-list)))
```

Comment:
With https://github.com/bdarcus/bibtex-actions/pull/199#discussion_r683958811 , the change is not needed.

Reply:
I don't understand; what's "this change"? The one I made, or the one you are suggesting?

Are you suggestion I resolve this without committing it?

Reply:
I mean the change to `bibtex-actions-run-default-action` is not needed if `cdr` is used in `bibtex-actions-dwim`. The final code is:

```lisp
(defun bibtex-actions-run-default-action (keys)
  "Run the default action `bibtex-actions-default-action' on KEYS."
  (funcall bibtex-actions-default-action
           (if (stringp keys) (split-string keys " & ") keys)))

;;;###autoload
(defun bibtex-actions-dwim ()
  "Run the default action on citation keys found at point."
  (interactive)
  (if-let ((keys (cdr (bibtex-actions-citation-key-at-point))))
      (bibtex-actions-run-default-action keys)))

```